sw/device/lib/dif/dif_entropy_src_unittest.cc

```diff
@@ -110,7 +141,7 @@ TEST_P(ConfigTestAllParams, ValidConfigurationMode) {
           {ENTROPY_SRC_CONF_BOOT_BYPASS_DISABLE_OFFSET, 0x5},
           // Current dif doesn ot set these fields
```

Comment:
nit: should this comment be removed or updated?

Reply:
Done.